extensions/barrage/src/main/java/io/deephaven/extensions/barrage/chunk/array/BooleanArrayExpansionKernel.java

```diff
@@ -40,15 +40,14 @@ public <A extends Any> WritableChunk<A> expand(
         final ObjectChunk<boolean[], A> typedSource = source.asObjectChunk();
 
         long totalSize = 0;
-        for (int ii = 0; ii < typedSource.size(); ++ii) {
-            int rowLen;
-            if (fixedSizeLength != 0) {
-                rowLen = Math.abs(fixedSizeLength);
-            } else {
+        if (fixedSizeLength != 0) {
+            totalSize = typedSource.size() * (long) fixedSizeLength;
```

Comment:
Does this need an absolute value on fixedSizeLength (and below on 98), the original code from before this change had total `rowLen = Math.abs(fixedSizeLength)`?

Reply:
Those Math.abs previously were superfluous. Only the ListChunkWriter needs to know about the head/tail restriction.. so I had it wrong the first time around.